extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -686,7 +686,10 @@ export class AmpSlideScroll extends BaseSlides {
    */
   showSlide_(newIndex) {
     const {noOfSlides_} = this;
-    newIndex = dev().assertNumber(newIndex);
+    if (isNaN(newIndex)) {
```

Comment:
From the stack trace, this looks like we're fixing the issue in the wrong place. We're getting a `NaN` from `getNextSlideIndex_` (called from `updateOnScroll_`).

Reply:
Thanks for the catch. I'll add the check there as well.